tests/usecase/test_db_config_options.py

```diff
@@ -0,0 +1,85 @@
+import os
```

Comment:
I would run `black` on this file. We don't enforce formatting for tests, but some of this could do with a black run.

Reply:
actually black is run on the test files. what exactly do you mean here?

Reply:
Eh, ignore this, a few things looked wrong to my eye, but actually were not. Never mind!